ext/droption/droption.h

```diff
@@ -46,6 +46,8 @@
 #define TESTALL(mask, var) (((mask) & (var)) == (mask))
 #define TESTANY(mask, var) (((mask) & (var)) != 0)
 
+#define DEFAULT_VALUE_SEP " "
```

Comment:
Let's put a prefix on this to avoid conflicts: DROPTION_DEFAULT_VALUE_SEP or DROPTION_DEFAULT_SEP or sthg